doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md

```diff
@@ -204,6 +204,7 @@ As seen above, the default configuration distributed with the image is tied to c
     * The user for Mongo and STH is `mongo` but can be changed through the CYGNUS_MONGO_USER environment variable.
     * The pass for Mongo and STH is `mongo` but can be changed through the CYGNUS_MONGO_PASS environment variable.
     * CYGNUS_MONGO_AUTH_SOURCE: Mongo option to specify which common user database source to use for authentication. Usually could be `admin`.
+    * CYGNUS_MONGO_REPLICA_SET: Mongo option to specify the replica set name. Usually could be `replica`.
```

Comment:
```suggestion
    * CYGNUS_MONGO_REPLICA_SET: Mongo option to specify the replica set name. Note that this env var is optional because Cygnus is able to connect to a MongoDB replica set without needing to specify its name.
```

Reply:
Fixed in cba084e